library/core/src/main/java/com/google/android/exoplayer2/metadata/dvbsi/AitDecoder.java

```diff
@@ -0,0 +1,195 @@
+package com.google.android.exoplayer2.metadata.dvbsi;
+
+import com.google.android.exoplayer2.metadata.Metadata;
+import com.google.android.exoplayer2.metadata.MetadataDecoder;
+import com.google.android.exoplayer2.metadata.MetadataInputBuffer;
+import com.google.android.exoplayer2.util.ParsableByteArray;
+import com.google.android.exoplayer2.util.TimestampAdjuster;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+
+public class AitDecoder implements MetadataDecoder {
+    // Specification of AIT can be found in 5.3.4 of TS 102 809 v1.1.1
```

Comment:
If this is meant to be explaining the `DESCRIPTOR_TRANSPORT_PROTOCOL` constant, it seems like 5.3.6 is a more precise reference?

I think we should add similar references to the other constants too to point to the relevant sections of the spec

Reply:
Well no I really meant to link this for global AIT documentation
I updated the documentation structure to make more sense